packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -16,6 +18,7 @@ import {
   FieldLabel,
 } from 'netlify-cms-ui-default';
 import { stringTemplate, validations } from 'netlify-cms-lib-widgets';
+import { ControlHint } from 'netlify-cms-core/src/components/Editor/EditorControlPane/EditorControl';
```

Comment:
Individual widgets don't reference the CMS core.

To use the `ControlHint` in a widget you'll need to extract that component to `netlify-cms-ui-default`

Reply:
Hey, thanks for the review. I will look into it sometime this week. Thanks.